specs/Modal.spec.js

```diff
@@ -234,25 +244,42 @@ describe('Modal', () => {
     const modal = renderModal({ isOpen: true });
     expect(modal.portal.content.style.position).toEqual(newStyle);
     Modal.defaultStyles.content.position = previousStyle;
+    unmountModal();
+  });
+
+  it('should remove class from body when no modals opened', () => {
+    const expectedBodyClass = 'ReactModal__Body--open';
+    const classList = document.body.classList;
+    renderModal({ isOpen: true });
+    renderModal({ isOpen: true });
+    expect(classList.contains(expectedBodyClass)).toBe(true);
+    unmountModal();
+    expect(classList.contains(expectedBodyClass)).toBe(false);
```

Comment:
We have pretty strict guidelines around modals not being stackable, but I understand that it is not a universal guideline, so keeping the component flexible makes sense.

I'll likely continue to use the `shouldComponentUpdate` approach within my composition of the component, but feel free to close if it's not the direction you're looking for 👍